.github/workflows/haskell.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7", "9.2.5"]
+        ghc: ["8.10.7", "9.2.7", "9.6.1"]
```

Comment:
Do we need to build all three of these?

Reply:
For now we do.

As soon as the performance of the `ghc-9.2` build has be validated by QA we can drop `ghc-8.10`.